ideas42ventures/ideas42ventures.com
# Adds new image card macro and content to about page

Fixes #32 

- Adds content to the about page.
- Creates an image card macro to reuse. 

**How to test:** 

1. Navigate to this [link](https://deploy-preview-36--ideas42ventures.netlify.app/about/) where there should be image placeholders. 
2. Cross-reference that with this [Figma doc ](https://www.figma.com/file/5ThSdBlZBcQnuyk15DWJNQ/Venture-Studio---Site?node-id=585%3A49)to ensure everyone is accounted for.
